web/router.ex

```diff
@@ -15,6 +15,7 @@ defmodule Tudo.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
+    post "/", PageController, :search
```

Comment:
I think this should still be a `get` request if you want to ensure the http verbs keep semantic meaning.
Perhaps look at using query params in the `index` or a different `search` controller